Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -32,8 +32,30 @@ enum ReaderModeTheme: String {
 }
 
 enum ReaderModeFontType: String {
+    private var isBoldFontEnabeld: Bool {
```

Comment:
Since you are using its value (`UIAccessibility.isBoldTextEnabled`) to initialize `self` and because this property will not be accessible at this moment, I think you could remove it. Also, you are not using it anywhere.